docset/winserver2019-ps/dnsserver/Set-DnsServerResourceRecord.md

```diff
@@ -32,7 +32,7 @@ This cmdlet cannot change the Name or Type of a DNS server resource record objec
 ### Example 1: Change the timespan of a resource record
 ```
 PS C:\> $OldObj = Get-DnsServerResourceRecord -Name "Host01" -ZoneName "contoso.com" -RRType "A"
-PS C:\> $NewObj = $OldObj.Clone()
+PS C:\> $NewObj = [ciminstance]::new($OldObj)
```

Comment:
```suggestion
PS C:\> $NewObj = [CimInstance]::new($OldObj)
```

Reply:
I believe [ciminstance] is actually the correct casing, since it is a [Type Accelerator](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_type_accelerators) just like [adsi], [hashtable], [int], [version], [wmi] etc.

Reply:
ok, agreed, thanks!